app/src/main/java/com/napzak/market/config/messaging/NapzakFirebaseMessaging.kt

```diff
@@ -10,10 +10,33 @@ import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import com.google.firebase.messaging.RemoteMessage
 import com.napzak.market.R.drawable.ic_push_notification
+import com.napzak.market.local.datastore.FirebaseServiceEntryPoint
+import com.napzak.market.local.datastore.TokenDataStore
+import com.napzak.market.notification.usecase.UpdatePushTokenUseCase
 import com.skydoves.firebase.messaging.lifecycle.ktx.LifecycleAwareFirebaseMessagingService
+import dagger.hilt.android.EntryPointAccessors
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 import timber.log.Timber
 
+private const val TAG = "FCM - okhttp"
+
 class NapzakFirebaseMessaging : LifecycleAwareFirebaseMessagingService() {
+
+    lateinit var dataStore: TokenDataStore
+    lateinit var updatePushTokenUseCase: UpdatePushTokenUseCase
+
+    override fun onCreate() {
+        super.onCreate()
+        val entryPoint = EntryPointAccessors.fromApplication(
+            applicationContext,
+            FirebaseServiceEntryPoint::class.java
+        )
+        dataStore = entryPoint.dataStore()
+        updatePushTokenUseCase = entryPoint.updatePushTokenUseCase()
+    }
```

Comment:
p2: `@AndroidEntryPoint` 어노테이션을 클래스에 붙이고 `@Inject`로 dataStore랑 유즈케이스를 주입할 수 있지 않나요??

```suggestion
@AndroidEntryPoint
class NapzakFirebaseMessaging: LifecycleAwareFirebaseMessagingService() {
      @Inject
      private lateinit var dataStore: TokenDataStore
      
      @Inject
      private lateinit var updatePushTokenUseCase: UpdatePushTokenUseCase

```

Reply:
이 부분은 지금도 오류가 뜨나요?